src/java.base/share/man/java.1

```diff
@@ -4005,6 +3997,15 @@ Example:
 \f[CB]\-Xlog:gc:garbage\-collection.log\f[R]
 .RE
 .TP
+.B \f[CB]\-XX:+ExtendedDTraceProbes\f[R]
+Deprecated. Use combination of these flags instead: -XX:+DTraceMethodProbes, -XX:+DTraceAllocProbes, -XX:+DTraceMonitorProbes
```

Comment:
Delete "Deprecated" as we are in the deprecated options section.

The wording also needs updating as per the warning text ... though that might read a little odd here so I suggest a tweak:
```
Use the combination of -XX:+DTraceMethodProbes, -XX:+DTraceAllocProbes and -XX:+DTraceMonitorProbes instead of this deprecated flag.
```

I would also move that new text to the end, so we still describe the flag first (otherwise it again reads a little odd.)

We will also need to add those flags to the "ADVANCED SERVICEABILITY OPTIONS FOR JAVA" section.

Reply:
@dholmes-ora thanks for the suggestions, I implemented them.